prelude/prelude.purs

```diff
@@ -120,6 +120,9 @@ module Prelude
   class Functor f where
     (<$>) :: forall a b. (a -> b) -> f a -> f b
 
+  fmap :: forall f a b. (Functor f) => (a -> b) -> f a -> f b
+  fmap = (<$>)
```

Comment:
Both good points. `map` would overlap with `Data.Array`, which is not the end of the world and would just necessitate a version bump.

Moving to `fmap` as the class default would break a lot of existing code though.